src/main/java/karuna/karuna_backend/email/MailjetEmailSender.java

```diff
@@ -0,0 +1,51 @@
+package karuna.karuna_backend.email;
+
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.resource.Emailv31;
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+class MailjetEmailSender implements EmailSender {
+
+    @Override
+    public void sendEmail(String to, String subject, String contentText, String contentHtml) {
+        try {
+            MailjetClient client = buildMailjetClient();
+            MailjetRequest request = configureRequest(to, subject, contentText, contentHtml);
+            client.post(request);
+            log.info("Success sent message!");
+        } catch (MailjetException ex) {
+            log.error("Error while sending email", ex);
+        }
```

Comment:
Obviously we need to let user know that something went wrong on frontend, so it would be good to handle this error in Controller. I think usual way.

Reply:
I make custom error and add handling